keras/utils/layer_utils.py

```diff
@@ -26,19 +26,19 @@ def print_summary(model, line_length=None, positions=None):
                 break
 
     if sequential_like:
-        line_length = line_length or 65
-        positions = positions or [.45, .85, 1.]
+        line_length = line_length or 80
+        positions = positions or [.40, .72, .85, 1.]
```

Comment:
The papers that report op count are those that are trying to make a point about op count efficiency, which is fairly niche. Most DL papers barely report their model architectures...